scripts/management/update_mgmt_db.py

```diff
@@ -1,119 +0,0 @@
-"""
-@author: jam335 - jason.motha@canterbury.ac.nz
-
-A script that updates a slurm mgmt db and updates the status of a task
-"""
-
-import argparse
-import qcore.constants
-from scripts.management import db_helper
-import qcore.constants as const
-
-
-def update_db(db, process, status, job=None, run_name=None, error=None):
```

Comment:
I think we need to restore a minimal version of this file

Reply:
Do we use this anywhere in our workflow code? Or only for manual updates?

Reply:
This is only used for manual updates. The alternative method is to run add_to_mgmt_queue while queue_monitor is running. 
This should also have a diff for the file I re added.